cdc/tests/integration_tests/cli_tls/run.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash
+
+set -eu
+
+CUR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $CUR/../_utils/test_prepare
+WORK_DIR=$OUT_DIR/$TEST_NAME
+CDC_BINARY=tikv-cdc.test
+SINK_TYPE=$1
+TLS_DIR=$(cd $CUR/../_certificates && pwd)
+UP_PD=https://$TLS_PD_HOST:$TLS_PD_PORT
+DOWN_PD=http://$DOWN_PD_HOST:$DOWN_PD_PORT
```

Comment:
Suggest to test on downstream with TLS. It's very likely that user will setup TLS for both upstream & downstream.

Reply:
test on downstream with TLS has added [sink_tls](https://github.com/tikv/migration/tree/main/cdc/tests/integration_tests/sink_tls), PTAL~

Reply:
I mean setup TLS for both upstream & downstream at this case.
IMO users are more likely to enable TLS for both at the same time.

Reply:
I also add  test for downstream with tls in `cli_tls` case